### PR TITLE
Showcase Paragraph Block FE

### PR DESCRIPTION
**Pull request by chris-lawton**
[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1348056831)

### Description of Changes Made

- Adds FE for the showcase paragraph block - i've taken the template work from https://github.com/torchbox/torchbox.com/pull/46
- Adds 2 SVG rings that are used in the hover state

Also fixes a double `:focus:focus` issue when using the `@include focus-styles` mixin.

### How to Test

In the pattern library:
- On it's template page: http://localhost:8000/pattern-library/render-pattern/patterns/molecules/streamfield/blocks/showcase_block.html
- On the components page: http://localhost:8000/pattern-library/render-pattern/patterns/styleguide/components/components.html

### Screenshots

Note - spacing around the component will be resolved when it's added to a page template

<details>
  <summary>Expand to see more</summary>

Screenshot 2024-01-23 at 16 01 31

Screenshot 2024-01-23 at 16 01 37

</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [ ] Tests and linting passes.

#### Unit tests

- [ ] Added
- [x] Not required

#### Documentation

- [ ] Updated build docs
- [ ] Updated editor guidelines (https://docs.google.com/document/d/1PAWccdQ4tfaZsrEWmpDhvP3GH5RRmBOARFVp4b-kje8/edit?usp=sharing)
- [x] Not required

#### Browser testing

- [x] I have tested in the following browsers and environments (edit the list as required)
  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android
- [ ] Not required

#### Data protection

- [x] Not relevant
- [ ] This adds new sources of PII and documents it and modifies Birdbath processors accordingly

#### Accessibility

- [x] Automated WCAG 2.1 tests pass
- [x] Manual WCAG 2.1 tests completed
- [x] I have tested in a screen reader
- [x] I have tested in high-contrast mode
- [x] Any animations removed for prefers-reduced-motion
- [ ] Not required

#### Sustainability

- [ ] Images are optimised and lazy-loading used where appropriate
- [x] SVGs have been optimised
- [ ] Perfomance and transfer of data considered
- [ ] If JavaScript is needed alternatives have been considered
- [ ] New font variants have not been added
- [ ] Not required

#### Pattern library

- [x] The pattern library component for this template displays correctly, and does not break parent templates
- [x] The styleguide is updated if relevant
- [ ] Changes are not relevant the pattern library

### CODE REVIEW

File: tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/showcase_block.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      - heading: Wagtail developer training
+        summary: Our training programme to build your understanding and skills in essential Django and Wagtail CMS practices.
+      - page: '#'
+        heading: 24/7 application support & hosting
+        summary: 24/7 application support and scalable, performant and resilient cloud hosting optimised for Wagtail.
+
+tags:
+  pageurl:
+    item.page:
+      raw: '#'
```
Comment (helenb) [inline tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/showcase_block.yaml:24]: Could you add in a `showcase_block.md` file and add some details about how these are expected to be used? We would not expect them to be used in practice with some items as links and some as not within the same showcase set - and that it is only done here as a demonstration. You also should update the editor guidelines to give details on this one.

**Review by helenb — APPROVED** [review]
Thanks - this is looking great. I did wonder if we ought to only link the heading, so the text below can be read out, but not a show-stopper for now. I think the only important thing is to update the editor guidelines for this one.

File: tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/showcase_block.yaml (hunk 1 above)
Comment (chris-lawton) [reply tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/showcase_block.yaml:24]: Added in https://github.com/torchbox/torchbox.com/pull/50/commits/6d552f4975d77a5da0659f2d0dc7556fa8246422 and also added a 'Streamfields' section in the editor guidelines

Comment (helenb) [reply tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/showcase_block.yaml]: also added an extra sentence in the editor's guidelines to make clear why it is important to keep them consistent.

**Pull request merged**